gap/congruences/conginv.gi

```diff
@@ -490,16 +490,16 @@ SEMIGROUPS.KernelTraceClosure := function(S, kernel, traceBlocks, pairstoapply)
   od;
   nrk := Length(kernelgenstoapply);
   Elements(kernel);
```

Comment:
What's this `Elements(kernel);` here for? I realise that this is not a line that you changed here, but this puzzled me when I looked at this function earlier too (see my comment in line 473 of this file).